mvpa2/datasets/sources/native.py

```diff
@@ -0,0 +1,110 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Loaders for PyMVPA's own demo datasets"""
```

Comment:
@yarikoptic Yeah, I was debating that with myself. pymvpa_demo is not good: sure it is pymvpa, all of this is. demo alone is not good, anything could be a demo. Hence, I settled with native. These are dataset source, this is our native one.

I was quite pleased with myself to come up with this reasoning.